### ISSUE TITLE
Drug Ontology

### ISSUE THREAD

**Issue by sandroacoelho** [sandroacoelho/drugs#2]
As a user, I want to have all medicine leaflets under an Ontology

**Comment by sandroacoelho** [comment]
@gcpdev : the leaflets have sections that describe components, diseases, and side-effects. 
I have split it in tables as you can see at the image. Maybe, we can write down some script to fill in that tables and then run an experiment, to annotate it and check what we achieve.


![model](https://user-images.githubusercontent.com/1160826/46911195-ab0afc00-cf2a-11e8-99d4-1aad23525a92.png)


I don't know if you agree, but we have almost all information to define an ontology.

**Comment by gcpdev** [comment]
@sandroacoelho yes, I do agree. We can start to build our ontology now. I'll look for compatible ontologies available in the LOD Cloud and so on.
